hoot-core/src/main/cpp/hoot/core/io/OsmGeoJsonWriter.cpp

```diff
@@ -180,6 +180,15 @@ void OsmGeoJsonWriter::_writeGeometry(const vector<long> &nodes, string type)
     if (_map->getNode(*it).get() != NULL)
       temp_nodes.push_back(*it);
   }
+  //  Empty nodes list should output an empty coordinate array
+  if (temp_nodes.size() == 0)
+  {
+    _writeKvp("type", type.c_str());
+    _write(",");
+    _write("\"coordinates\": []");
+    return;
+  }
+  //  Update the geometry type if necessary
```

Comment:
While trying to convert the data from the commandline I found that empty geometries weren't being written to GeoJSON correctly so I fix it here.  Then I later learned that you were using the `osmtogeojson` library and this wasn't the cause of the issue.  Still needed to be fixed.